src/tokens/Assets.tsx

```diff
@@ -414,6 +415,19 @@ function AssetsScreen({ navigation, route }: Props) {
     return items
   }, [t, displayPositions])
 
+  useLayoutEffect(() => {
```

Comment:
nit: while the order in which we declare variables and hooks don't affect the outcome at runtime, components can get long and difficult to read so it helps to have a predictable order. generally i think we're using the pattern of declaring state and hooks at the top of the component, and handlers / variables after those. can we please move this function to live closer to the other hooks?